how-we-think/index.html

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+---
+{% include header5.html %}
+    <div class="container">
+        <style type="text/css" scoped>
+            h1.title {
+                text-align: center;
+                margin: 0 0 50px;
+                font-family: 'Libre Baskerville', Baskerville, "Baskerville Old Face", "Hoefler Text", Garamond, "Times New Roman", serif;
+            }
+            section.main-text {
+                background: white;
+                padding: 50px 14%;
+                margin-bottom: 100px;
+            }
+            @media screen and (max-width: 600px) {
+                section.main-text {
+                    padding: 50px 20px;
+                }
+            }
+
+        </style>
+        <section class="main-text">
+            <h1 class="title">how we think</h1>
+            <p>
+                Our <b>core values and goals</b> are ones of:
+            </p>
+                <ul>
+                    <li>Diversity</li>
+                    <li>Inclusion</li>
+                    <li>Fair and safe working conditions</li>
+                    <li>Working together across disciplines and departments</li>
+                </ul>
+            <p>
+            And we believe that these are necessary in creating a <b>pleasant, sustainable, and good working place</b> and
+            in creating great software products.
+            </p>
+            <p>
+                One of the main problems in workplaces today, especially when it comes to IT, is that the people who
+                are supposed to be working together <b>do not understand what each other are doing or their reasoning behind it</b>.
```

Comment:
Suggestions:

"what each other" -> "what the others"

"or their reasoning behind it" -> "or the reasoning behind their actions"